api/v1alpha1/ratelimit_types.go

```diff
@@ -28,8 +28,11 @@ type RateLimitSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of RateLimit. Edit RateLimit_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	Conditions []string `json:"conditions"`
+	MaxValue   int      `json:"max_value"`
+	Namespace  string   `json:"namespace"`
```

Comment:
All k8s objects have a meta property `namespace` you can use to bind one rate limit object to the limitador service of the same namespace. JFYI.

Reply:
The naming is confusing, sorry. A limitador namespace doesn't need to be a kubernetes namespace. Limitador also runs outside kubernetes. A limitador namespace is the "context" on which a limit applies, it could be a service, a product, a proxy, a kubernetes namespace or any other thing really. It's meant to be generic to support multiple use cases.

In a future version of Limitador we might want to rename this field so it's less confusing for users running it in Kubernetes.

Reply:
Ok, I see, it is an internal namespace.